tools/wpt/commands.json

```diff
@@ -52,8 +52,8 @@
     "parser": "get_parser",
     "help": "Install browser components",
     "install": [
-      "mozdownload",
-      "mozinstall"
+	    "mozdownload",
+	    "mozinstall"
```

Comment:
undo all the whitespace changes in this file please. We don't want to muddy-up the git history